apps/emqx_connector/src/emqx_connector_http.erl

```diff
@@ -601,6 +603,15 @@ is_sensitive_key(_) ->
 redact(Data) ->
     emqx_misc:redact(Data, fun is_sensitive_key/1).
 
+%% because the body may contain some sensitive data
+%% and at the same time the redact function will not scan the binary data
+%% and we also can't know the body format and where the sensitive data will be
+%% so the easy way to keep data security is redacted the whole body
+redact_request({_Path, _Headers} = Req) ->
```

Comment:
Should we redact headers either? Would sensitive data be exposed if a
some user connects to http resources using some header-based authentication?

Reply:
No, it isn't necessary because the `redact` will scan the headers